_layouts/default.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8">
-    <title>{{ page.title | default: site.title }}</title>
+    <title>{{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}</title>
```

Comment:
Can we add `{% if page.title %}{{ page.title }} | {% endif %}` before the site title? That way pages with explicit titles will still be distinct? If so, I think this can be merged. Sorry for the delay here noodling if the theme should be for single page sites, multi-page sites, or both. I think the above strikes a happy opt-in balance.